app/models/check_in.rb

```diff
@@ -15,7 +15,7 @@ def log_entries_attributes=(log_entries_attributes)
 
   def log_entries
     Activity.active.map do |activity|
```

Comment:
We want to restrict this to the user for the check-in's active activities; not *all* the activities.